tx/transaction.go

```diff
@@ -28,9 +28,10 @@ var (
 	errEmptyTypedTx         = errors.New("empty typed transaction bytes")
 )
 
+// Starting from the max value allowed to avoid ambiguity with Ethereum tx type codes.
 const (
-	LegacyTxType     = 0x00
-	DynamicFeeTxType = 0x01
+	LegacyTxType     = 0x7f
```

Comment:
Should it be `0xff` ?

Reply:
I'm taking as a reference the [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718#transactiontype-only-goes-up-to-0x7f) where the max value used for tx type is `7f` since all legacy txs will start with `0x80` at least, and we are the same